torch/_refs/linalg/__init__.py

```diff
@@ -57,6 +66,25 @@ def _check_norm_dtype(dtype: Optional[torch.dtype], x_dtype: torch.dtype, fn_nam
 from torch._decomp import register_decomposition
 
 
+@register_decomposition(torch._ops.ops.aten.linalg_cross)
+@out_wrapper()
+def cross(a: Tensor, b: Tensor, dim: int = -1):
```

Comment:
Given the CI failures I say we should just use opmath. There's no reason we shouldn't want higher precision than eager.